playgrounds/remix/app/routes/_index.tsx

```diff
@@ -0,0 +1,17 @@
+import type { MetaFunction } from '@remix-run/node';
+import { Button } from '@itwin/itwinui-react';
+
+export const meta: MetaFunction = () => {
+  return [
+    { title: 'New Remix App' },
+    { name: 'description', content: 'Welcome to Remix!' },
```

Comment:
nit: You can change this meta information if you want.

Reply:
Okay, will do.